packages/site/mosaic.config.js

```diff
@@ -1,9 +1,14 @@
 const deepmerge = require('deepmerge');
 const mosaicConfig = require('@jpmorganchase/mosaic-standard-generator/dist/fs.config.js');
 
-module.exports = deepmerge(mosaicConfig, {
-  deployment: { mode: 'snapshot-file', platform: 'vercel' },
+const siteConfig = {
+  ...mosaicConfig,
   plugins: [
+    ...mosaicConfig.plugins,
+    {
+      modulePath: '@jpmorganchase/mosaic-plugins/SidebarPlugin',
```

Comment:
this should be set in the standard generator config rather than overriden.
The defaults should match what Salt needs, I would propose we design the Mosaic site with the same sidebar IA (or as close as possible).